templates/events/select_status.html

```diff
@@ -64,4 +64,16 @@
 
     $("#from, #to").on('change', reselect_time_range)
     $("#time-range-reset").on('click', reselect_time_range)
+    $("#per_page").on('change', function () {
+        $.ajax({
+            url: "{{ url_for('user_events.per_page') }}",
+            dataType: 'json',
+            type: 'POST',
+            data: {
+                per_page: document.getElementById('per_page').value,
+            },
+        }).always(function () {
+            location.reload(true);
```

Comment:
maybe another solution here. get the value of events number per page and call the endpoint https://github.com/rokwire/events-manager/blob/develop/user_events.py#L39 to render the page.